xtras/src/address_map.rs

```diff
@@ -49,6 +50,8 @@ where
         self.inner.insert(key, address);
     }
 
+    // TODO: Support messages with a return value, but that might be out of scope for this (add
+    // ticket)
```

Comment:
perhaps we could just add a ticket for this one? (this can then be discussed/acted on/dropped separately). Not sure if it's a part of this changeset at all.